test/utils/login_page/credentials.js

```diff
@@ -0,0 +1,4 @@
+const login = "my_login";
+const password = "my_password";
+
+module.exports = {login, password};
```

Comment:
Please move `credentials.js` folder to a separate `constants` folder under `test/utils/constants` path.

Reply:
done